### PR TITLE
add named accessor documentation

### PR DESCRIPTION
**Pull request by CrowdHailer**
Thought it would be good to have this complete the 1.15 milestone

https://github.com/gleam-lang/gleam/issues/1031

Note I don't have ruby installed so haven't done the generation of the book.

### CODE REVIEW

File: book-src/tour/custom-types.md [hunk 1]
```diff
@@ -102,6 +102,18 @@ let Points(p) = score
 p // => 50
 ```
 
+## Named accessors
+
+If a custom type has only one varient and named fields they can be accessed
```
Comment (tranzystorek-io) [inline book-src/tour/custom-types.md:107]: variant

Comment (CrowdHailer) [reply book-src/tour/custom-types.md:107]: thanks, are you able to write suggestions to PR's or is that only louis?

Comment (CrowdHailer) [reply book-src/tour/custom-types.md:107]:
```suggestion
If a custom type has only one variant and named fields they can be accessed
```

**Review by lpil — APPROVED** [review]
Thank you! I've been super busy this week so this is really helpful 💜

**Comment by CrowdHailer** [comment]
Not at all. I've been really enjoying contributing.

I hope it's busy in a good way

**Pull request merged**